src/AbstractSQLRules2SQL.ts

```diff
@@ -996,6 +1010,14 @@ const typeRules: Dictionary<MatchFn> = {
 			return `TIME(${date})`;
 		}
 	},
+	ToJSON: (args, indent) => {
+		checkMinArgs('ToJSON', args, 1);
+		if (engine !== Engines.postgres) {
+			throw new SyntaxError('ToJSON not supported on: ' + engine);
+		}
+		const column = TextValue(getAbstractSqlQuery(args, 0), indent);
```

Comment:
This looks like it can be any value, at least according to postgres docs https://www.postgresql.org/docs/14/functions-json.html#FUNCTIONS-JSON-CREATION-TABLE `to_json ( anyelement ) → json`
```suggestion
		const column = AnyValue(getAbstractSqlQuery(args, 0), indent);
```